frontend/openchat-client/src/openchat.ts

```diff
@@ -936,6 +938,38 @@ export class OpenChat extends EventTarget {
         return this.#liveState.isDiamond ? DIAMOND_MAX_SIZES : FREE_MAX_SIZES;
     }
 
+    buildIdenticonUrl(userId: string): string {
```

Comment:
Is this actually used?

Reply:
Uhm no - not sure what I was doing there.